EasyVCR/CensorElement.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+
+namespace EasyVCR
+{
+    public class CensorElement
+    {
+        /// <summary>
+        ///     Whether the name is case-sensitive.
+        /// </summary>
+        private bool CaseSensitive { get; set; }
+        /// <summary>
+        ///     Name of the element to censor.
+        /// </summary>
+        private string Name { get; set; }
```

Comment:
Should not be capitalized

Reply:
What should not be capitalized?

Reply:
The variable names, shouldn't they be camel cases?

Reply:
That's a Java convention. In C#, it's TitleCase.